Documentation/kubernetes-on-aws.md

```diff
@@ -156,6 +158,20 @@ You can now customize your cluster by editing asset files:
 
 You can also now check the `my-cluster` asset directory into version control if you desire. The contents of this directory are your reproducible cluster assets. Please take care not to commit the `my-cluster/credentials` directory, as it contains your TLS secrets. If you're using git, the `credentials` directory will already be ignored for you.
 
+#### Route53 Host Record (optional)
+
+`kube-aws` can optionally create an A record for the controller IP in an existing hosted zone.
```

Comment:
As of now, no. When that time comes, there will be a few other things that will have to change as well.